### DISCUSSION TITLE
Copilot disappeared in the down tap it's like not installed , even i re install it

### DISCUSSION THREAD

**Discussion by Abdullamhd** [community/community#41956] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I can't use github copilot it's disappeared from the down tab its like not installed even i re install it multiple time but still not working 
Im using Visual studio code in WSL2 

--- 
Update : 

I Installed the previous version it's working  , version :v1.63.7601

but the last updated version is not working : 1.64.7694

**Comment by Kimrokgil** [comment]
me too!

**Reply by Abdullamhd** [reply]
Try to install this version  v1.63.7601
it's worked for me

**Comment by kavisek** [comment]
I had to downgrade the vscode github copilot extension version to 1.63.7601 get it to work. A version from 3 days ago.

**Reply by kavisek** [reply]
Downgrading VSCode did not work. Something is up with the new version of the extension. Downgrade the extension to 1.63.7601

**Reply by Abdullamhd** [reply]
I installed this version it's working now v1.63.7601 , 
but last version is not working

**Reply by kavisek** [reply]
Ya version 1.64.7694 which was released 5 hours ago is broken.

**Reply by conqr2** [reply]
Github Employees watching the forums after breaking CoPilot for the 3rd time this month: 😮

**Comment by iamcjwilliams** [comment]
@Abdullamhd Sorry for the trouble here, please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-visual-studio-code
